keyboards/zeal60/rules.mk

```diff
@@ -76,4 +76,6 @@ RAW_ENABLE = yes
 DYNAMIC_KEYMAP_ENABLE = yes
 CIE1931_CURVE = yes
 
-LAYOUTS = 60_ansi 60_iso 60_hhkb 60_ansi_split_bs_rshift
+# 60_iso  removed because unixmaal's community layout won't compile
+# due to duplicate function definition https://travis-ci.org/qmk/qmk_firmware/builds/472014965
+LAYOUTS = 60_ansi 60_hhkb 60_ansi_split_bs_rshift
```

Comment:
Right, I am caught in the middle here… No one would approve the pull request with build failures but the build failures come from code that I haven't touched and unrelated to my change. I will defer to your decision as the maintainer on wether or not  to support generic community layouts on this board. If your decision is to not support them I have no problem removing the `LAYOUTS` assignment here.

Reply:
>No one would approve the pull request with build failures but the build failures come from code that I haven't touched and unrelated to my change.

As long as the default keymaps compile, that's the really important part.  

> People seem OK with Travis failing due to the hazards of having user code and keyboard-level code conflict and neither side wanting to clean up the mess.

Because there isn't a good solution for how to handle this.  It is something that we've discussed a lot.  

As long as the default keymaps are compiling, then the board is good.   And it's been my general policy to flag the maintainers of the community layouts that are erroring out, so they can fix them. 


The other issue, is that we have no way to compile every board that supports a specific layout (eg `make LAYOUT_ortho_4x12:drashna` to compile all the boards that support that layout), so there is no way to know if your layout breaks something on another keymap.

And right now, the only issue the Zeal60 has with community layouts is @unxmaal's because it uses `action_function`, and so does the core Zeal60 code.   And since there is no `*_kb`/`*_user` level function for this (and I suspect that this is why `process_record_*` was created in the first place), it's erroring out. 

So, I'd rather that `LAYOUTS` was not modified here. 

```suggestion
LAYOUTS = 60_ansi 60_iso 60_hhkb 60_ansi_split_bs_rshift
```

Reply:
heh, well travis doesn't appear to be triggering the full build of all the keymaps any more so the builds are showing as passing. when I `make zeal60` locally I still get the errors from before. Perhaps we can put this one to rest for now?